demo/Makefile

```diff
@@ -0,0 +1,80 @@
+
+# Domain to use when starting flynn
+CLUSTER_DOMAIN := demo.localflynn.com
+CLUSTER := default
+BOOTSTRAP_LOG_FILE := bootstrap.${CLUSTER}.log
+
+# Providers: virtualbox, vmware_fusion, aws
+PROVIDER := virtualbox
+
+
+.PHONY: install-cli init bootstrap up down halt update ssh reset token destroy dashboard ssh-cmd
+
+
+# Install flynn cli on host
+install-cli:
+	L=/usr/local/bin/flynn && curl -sL -A "`uname -sp`" https://dl.flynn.io/cli | zcat >$L && chmod +x $L
+
+# Initialize a new cluster
+init:
+	@$(MAKE) up
+	@$(MAKE) bootstrap
+	flynn key add
+	@printf "\nFlynn is now running in the VM.\nOpen dashboard: make dashboard\n\n"
+
+# Bootstrap the cluster
+# This should only be called once per cluster
+# Runs bootstrap in VM and cluster add on host
+bootstrap:
+	@$(MAKE) CMD='CLUSTER_DOMAIN="${CLUSTER_DOMAIN}" flynn-host bootstrap /etc/flynn/bootstrap-manifest.json 2>&1' ssh-cmd | tee ${BOOTSTRAP_LOG_FILE}
+	@(grep '^flynn cluster add .*$$' ${BOOTSTRAP_LOG_FILE} || echo exit 1) | sed "s/$$(printf '\r')\$$//" | bash
```

Comment:
How do I set the name of the cluster? It seems like the `flynn-host bootstrap` command should include the cluster name since it outputs the command that's fed into bash. But I don't see a way in the help output to change the cluster name.

We could grep for `default` in the output but this seems kludgy.